docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -270,6 +270,10 @@ KMM will then create worker Pods where required to run `modprobe -r` and unload
     KMM ships with a validating admission webhook that rejects the deletion of namespaces that contain at least one
     `Module` resource.
 
+### Kernel modules events on Nodes
```

Comment:
lets change it to "some events may not...", otherwise it reads like all the events will be missing